backend/app/routers/providers.py

```diff
@@ -11,7 +12,12 @@
 )
 
 
-@router.get("/{country_code}", response_model=list[str])
+class Provider(BaseModel):
+    provider_name: str
+    display_priority: int
```

Comment:
Can we move this into schemas(models soon :tm:) :D

Reply:
Done